sway/desktop/render.c

```diff
@@ -311,8 +310,8 @@ void render_blur(bool optimized, struct sway_output *output,
 	pixman_region32_copy(&inverse_opaque, opaque_region);
 	pixman_region32_inverse(&inverse_opaque, &inverse_opaque, &surface_box);
 	pixman_region32_intersect_rect(&inverse_opaque, &inverse_opaque, 0, 0,
-			surface_width * surface_scale,
-			surface_height * surface_scale);
+			dst_box->width * surface_scale,
+			dst_box->height * surface_scale);
```

Comment:
Since `surface_box` is defined by `dst_box`, should we remove `surface_scale` as a param, and then remove the `intersect_rect` line? Seems weird that in surface_box we divide by the output scale, whereas here we multiply by the surface scale